server/tests/2-recipes.js

```diff
@@ -172,7 +172,7 @@ describe('/GET all Recipes by User Test', () => {
       .set('Accept', 'application/json')
       .set('x-access-token', token)
       .end((err, res) => {
-        expect(res.statusCode).to.equal(201);
+        expect(res.statusCode).to.equal(200);
```

Comment:
Why are you testing for only the status code here? You should test the data inside the JSON array that is returned, and make sure they are correct